src/assets/scss/global.scss

```diff
@@ -0,0 +1,5 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');
+@import './_reset.scss';
+@import './mixin.scss';
+
+iframe { display: none}
```

Comment:
This was only for your own use and please don't push it on git.

Reply:
Okay got it... thank you